nion/swift/ExportDialog.py

```diff
@@ -273,10 +277,31 @@ def __init__(self, display_item: DisplayItem.DisplayItem) -> None:
         self.__enforce_width_height_constraints()
 
     def __calculate_display_size_in_pixels(self, display_item: DisplayItem.DisplayItem) -> Geometry.IntSize:
-        if display_item.display_data_shape and len(display_item.display_data_shape) == 2:
-            return Geometry.IntSize(height=display_item.display_data_shape[0], width=display_item.display_data_shape[1])
+        if display_item.display_data_shape and self.__display_item.used_display_type != "line_plot":
+            return Geometry.IntSize(height=display_item.display_data_shape[-2], width=display_item.display_data_shape[-1])
         return Geometry.IntSize(height=288, width=480)
 
+    def __update_dimensional_calibrations(self, property: str) -> None:
+        """Recalculate the image information string based on updated scales"""
+        if property == "dimensional_calibrations" or property == "displayed_dimensional_calibrations":
+            self.notify_property_changed("image_info")
```

Comment:
Leave out for now. This is complicated and I think it's going to need a lot more work. Not only can dimensions change, but the title can change, etc. Let's leave this to another PR.